var/spack/repos/builtin/packages/py-bbp-morphology-workflow/package.py

```diff
@@ -17,6 +17,7 @@ class PyBbpMorphologyWorkflow(PythonPackage):
     depends_on('py-setuptools', type=('build', 'run'))
 
     depends_on('py-neuroc', type='run')
+    depends_on('py-scipy', type='run')
```

Comment:
`py-morphtool` already depends on this. Is it a direct dependency?

Reply:
Because it was still complaining that scipy was not found even though it was in the morph-tool recipe. But I think was Spack env was screwed.